code/modules/vehicles/wheelchair.dm

```diff
@@ -165,3 +190,25 @@
 /obj/item/wheelchair/Destroy()
 	wheelchair = null
 	. = ..()
+
+/obj/item/wheelchair/golden
+	name = "gilded wheelchair"
+	icon_state = "wheelchair_folded_gold"
+	custom_materials = list(/datum/material/gold = 20000)
+	var/obj/vehicle/ridden/wheelchair/golden/wheelchair2
```

Comment:
what the var doing

Reply:
the var doin nothing

thanks for pointing it out

Reply:
OH RIGHT

This var for some reason, somehow, controls which wheelchair is spawned when you *UN*fold the damned thing

Reply:
the original one does, this one does nothing

Reply:
since that var keeps track of the wheelchair inside the item

Reply:
I mean without that it somehow bricks and spawns a normal wheelchair...with that it works.